Changelog.md

```diff
@@ -1,3 +1,17 @@
+# 0.8.0.0
+
+- GHC 8.6.4 support added.
+- Resolver bumped, LTS 13.10 for GHC 8.6.3, LTS 13.15 for GHC 8.6.4 (@alanz)
+- Clarify install section of README.md (@Anrock, @antonlogvinenko)
```

Comment:
Lost in review interface of github, will repost just in case.
My changes to README removed any mentions of other build methods except Shake. So imo it would be better to credit this item to @antonlogvinenko only and add additional item like "Shake is now the only supported method of building HIE".

Reply:
Maybe combine it with item about removing Makefile and powershell script.